### ISSUE TITLE
Headers with Accept */*

### ISSUE THREAD

**Issue by kikoseijo** [Urigo/graphql-cli#315]
Can´t the Header `*/*` on the Accept be removed or overwritten.
Having issues on redirection to login when they exists on requests.

I have tried to create the customs, but `*/*` its always there and a Laravel server does not respond as expected.

```
{
  "Accept":"application/json",
  "Content-Type":"application/json"
}
```

Any help or trick it will be welcome.

**Comment by ardatan** [comment]
Currently we have new structure for GraphQL CLI and in the new alpha version `4.0.0-alpha.0`.
Headers can be defined inside [GraphQL Config](https://graphql-config.com/docs/schema) like below;
```yml
schema:
  - URL:
      headers:
         Authorization: [TOKEN]
```
Checkout new instructions and we will be happy to hear your feedback. Feel free to contact us using our Discord channel;
https://discord.gg/xud7bH

**Comment by Urigo** [comment]
We've just released a new version and an alpha release channel for GraphQL CLI - `4.0.0-alpha.XXX`!

We've also moved to use the new `alpha` version of [`graphql-config`](https://medium.com/the-guild/graphql-config-708dd0e5d15f)!

Headers can be defined inside [GraphQL Config](https://graphql-config.com/docs/schema) like below;
```yml
schema:
  - URL:
      headers:
         Authorization: [TOKEN]
```

Would that solve your issue?

Checkout new instructions and the [migration guide](https://github.com/urigo/graphql-cli#migration-from-3xx-to-4xx) and we will be happy to hear your feedback during the alpha period!

Feel free to contact us by opening an issue or using our Discord channel;
https://discord.gg/xud7bH
